src/Build/ElementLocation/ElementLocation.cs

```diff
@@ -209,7 +209,7 @@ internal static ElementLocation Create(string file, int line, int column)
         /// </summary>
         private static string GetLocationString(string file, int line, int column)
         {
-            string locationString = String.Empty;
```

Comment:
It's guaranteed to be set (without having been read) in one of the three clauses below. If this is changed such that the "else" doesn't exist, for instance, it would throw a compile-time error, so this is actually safer than initializing it to empty.